db/index.js

```diff
@@ -2,3 +2,37 @@ const mysql = require('mysql');
 const Config = require('./config.js');
 
 const connection = mysql.createConnection(Config);
+
+connection.connect(err => {
+  if (err) {
+    console.log(err);
+    return
+  }
+  console.log('connected to db');
+});
+
+// let getProductInfo = function(productId, cb) {
+//   connection.query(`SELECT * FROM products WHERE id LIKE ${productId}`, (err, info) => {
+//     if (err) {
+//       cb(err, null);
+//     } else if (info !== null) {
+//       cb(null, info);
+//     }
+//   })
+// }
+
+let getAllProductsOfStyle = function(style, cb) {
+  connection.query(`SELECT * FROM products WHERE style LIKE ${style}`, (err, products) => {
```

Comment:
I think you can use promisifyall for mysql and on your server you can do something like 'db.getAllProductsOfStyle.then(data => ...)'
Using cb for asynchronous function sometimes makes really difficult to find bugs in my opinion...